projects/person-search-index-from-delius/deploy/template/task-definition.yml

```diff
@@ -5,7 +5,7 @@ containerDefinitions:
   - name: {{ .values.image.name }}
     image: ghcr.io/ministryofjustice/hmpps-probation-integration-services/{{ .values.image.name }}:{{ .Env.VERSION }}
     healthCheck:
-      command: ["CMD-SHELL", "exit 0"]
+      command: ["CMD-SHELL", "curl -sf localhost:9600"]
```

Comment:
```suggestion
      command: ["CMD-SHELL", "curl -sSf localhost:9600"]
```

Reply:
Even if we show the error with `-S`, it'll get swallowed by ECS anyway won't it?  Happy to make the change though if you think it's required?

Reply:
For my own reference as I had to look it up 😄
 https://explainshell.com/explain?cmd=curl+-sSf+localhost%3A9600